.buildconfig.yml

```diff
@@ -147,6 +151,10 @@ projects:
     path: components/feature/downloads
     description: 'Feature implementation for apps that want to use Android downloads manager.'
     publish: true
+  feature-pocket-recommendations:
+    path: components/feature/pocket-recommendations
+    description: 'Feature implementation for apps that want to display Pocket recommended articles'
+    publish: true
```

Comment:
Does this require a separate component, since we also have `service-pocket`? I think ideally we have just one pocket component that provides all kinds of pocket API things (Think of a "pocket SDK")? :)